chatterbot/storage/sqlalchemy_storage.py

```diff
@@ -98,30 +99,19 @@ def __init__(self, **kwargs):
             "read_only", False
         )
 
-        self.drop_create = self.kwargs.get(
-            "drop_create", False
-        )
+        create = self.kwargs.get("create", False)
 
-        if not self.read_only and self.drop_create:
-            _base.metadata.drop_all(self.engine)
-            _base.metadata.create_all(self.engine)
+        if not self.read_only and create:
+            self.create()
+
+        Session = sessionmaker(bind=self.engine)
+        self.session = Session()
```

Comment:
Another sugestion is to check if session avaliable, is yes reuse , if not create new one